app/src/main/res/layout/activity_about.xml

```diff
@@ -49,12 +49,15 @@
 
                 android:columnCount="2"
                 android:padding="14dp"
-                android:rowCount="2">
+                android:rowCount="2"
+                >
```

Comment:
Please add **>** adjacent to that **rowCount** attribute value. No need of extra line is required